webapp/channels/src/components/analytics/true_up_review.test.tsx

```diff
@@ -25,7 +25,7 @@ describe('TrueUpReview', () => {
                     IsLicensed: 'true',
                 }),
                 config: {
-                    EnableDiagnostics: 'false',
+                    EnableDiagnostics: 'true',
```

Comment:
Can we add a test with telemetry disabled they still see the button?

Reply:
That test is here: https://github.com/mattermost/mattermost/pull/26073/files#diff-3655f7bd4881b8a98c4e058b55723afa0757a79c2d6a5a519e0dcf7e3dc22facR76

Reply:
Perfect, thanks!